dff/pipeline/pipeline/component.py

```diff
@@ -27,22 +32,21 @@ class PipelineComponent(abc.ABC):
     This class represents a pipeline component, which is a service or a service group.
     It contains some fields that they have in common.
 
-    :param before_handler: before handler, associated with this component
-    :param after_handler: after handler, associated with this component
-    :param timeout: (for asynchronous only!) maximum component execution time (in seconds),
-        if it exceeds this time, it is interrupted
-    :param requested_async_flag: requested asynchronous property;
-        if not defined, calculated_async_flag is used instead
-    :param calculated_async_flag: whether the component can be asynchronous or not
-
-        - for :py:class:`~pipeline.service.service.Service`: whether its ``handler`` is asynchronous or not
-        - for :py:class:`~pipeline.service.group.ServiceGroup`: whether all its ``services`` are asynchronous or not
-
-    :param start_condition: StartConditionCheckerFunction that is invoked before each component execution;
-        component is executed only if it returns True
-    :param name: component name (should be unique in single :py:class:`~pipeline.service.group.ServiceGroup`),
-        should not be blank or contain `.` symbol
-    :param path: separated by dots path to component, is universally unique.
+    :param before_handler: Before handler, associated with this component.
```

Comment:
Currently this is displayed this way:
![Screenshot_20221227_182820](https://user-images.githubusercontent.com/61429541/209687484-1d618862-a96c-402e-a73a-a86f23003638.png)
So a `:type:` directive should be added:
```suggestion
    :param before_handler: Before handler, associated with this component.
    :type before_handler: Optional[:py:data:`~.ExtraHandlerBuilder`]
```
And it would be displayed this way:
![Screenshot_20221227_182615](https://user-images.githubusercontent.com/61429541/209687576-19164f12-e3e2-4caf-9c11-86ed6863e076.png)

This needs to be fixed in other places as well.